Docs/source/Software/Tasks.rst

```diff
@@ -2,6 +2,31 @@
 Real Time Operating System Tasks
 ********************************
 
+Initialization Task: Alex Koo
+===============================
```

Comment:
Also include that it creates semaphores, why it creates those semaphores, why the order of the creation matters. Say that it deletes itself in at the end and that we save the contents of floating point registers whenever there is a context switch (this is one of the options in OSTaskCreate()).